dash/dash-config.cmake.in

```diff
@@ -8,14 +8,14 @@
 
 @PACKAGE_INIT@
 
-set(DASH_VERSION_MAJOR "@DASH_VERSION_MAJOR@")
-set(DASH_VERSION_MINOR "@DASH_VERSION_MINOR@")
-set(DASH_VERSION_PATCH "@DASH_VERSION_PATCH@")
+set(DASH_VERSION_MAJOR "@DASH_VERSION_MAJOR@")
+set(DASH_VERSION_MINOR "@DASH_VERSION_MINOR@")
+set(DASH_VERSION_PATCH "@DASH_VERSION_PATCH@")
 
 set(DASH_LIBRARY "@DASH_LIBRARY@")
 set(DASH_LIBRARIES ${DASH_LIBRARY} "dart-@dart_variant@")
 set(DASH_CXX_FLAGS "@VARIANT_ADDITIONAL_COMPILE_FLAGS@ @CMAKE_CXX_FLAGS_RELEASE@")
```

Comment:
With the changes I made to the config file, I recommend to use it. Many of the flags represent the cmake configuration of the dash build (e.g. use hwloc,...). So if not using them, you might run into problems as the headers might not fit to the installed library. However currently even the optimization flags are included. That is what I fixed in the referenced commit one comment above.